source/installation.html.haml

```diff
@@ -19,24 +19,37 @@ title: Installation
             Windows
           %p
             All binary packages are unofficial third-party builds.
+            Except for those linked from the mpv-player/mpv repository,
+            each commit and pull request has associated test builds.
 
-      = package_row 'Windows builds by shinchiro (releases and git)',
-                    'https://sourceforge.net/projects/mpv-player-windows/files',
+      = package_row 'Windows builds by shinchiro (git)',
+                    'https://github.com/shinchiro/mpv-winbuild-cmake/releases',
                     :"cloud-download"
-      = package_row 'Scoop', 'https://github.com/lukesampson/scoop-extras/blob/master/bucket/mpv.json'
-      = package_row 'Scoop (git)', 'https://github.com/lukesampson/scoop-extras/blob/master/bucket/mpv-git.json'
+      = package_row 'Windows builds by zhongfly (git)',
+                    'https://github.com/zhongfly/mpv-winbuild/releases',
+                    :"cloud-download"
+      = package_row 'First-party test builds on GitHub Actions',
+                    'https://github.com/mpv-player/mpv/actions/workflows/build.yml?query=event:push+branch:master',
+                    :"cloud-download"
+      = package_row 'Scoop', 'https://github.com/ScoopInstaller/Extras/blob/master/bucket/mpv.json'
```

Comment:
we could link this instead: https://scoop.sh/#/apps?q=mpv&id=b05b47128464d8969416289383fbfc69a47353e3

Reply:
changed and also did the same for MSYS2 link